backend/grpc-server/src/server/payments.rs

```diff
@@ -618,6 +643,133 @@ impl Payments {
             )),
         }
     }
+
+    async fn handle_payment_session_token<
+        T: PaymentMethodDataTypes
+            + Default
+            + Eq
+            + Debug
+            + Send
+            + serde::Serialize
+            + serde::de::DeserializeOwned
+            + Clone
+            + Sync
+            + domain_types::types::CardConversionHelper<T>,
+    >(
+        &self,
+        connector_data: ConnectorData<T>,
+        payment_flow_data: &PaymentFlowData,
+        connector_auth_details: ConnectorAuthType,
+        event_params: EventParams<'_>,
+        payload: &PaymentServiceAuthorizeRequest,
+    ) -> Result<PaymentMethodTokenResponse, PaymentAuthorizationError> {
```

Comment:
This handler should have its own error, the error will be transformed into PaymentAuthorizationError in PaymentAuthorize flows